README.md

```diff
@@ -1,69 +1,41 @@
+# build-ci
+
 A central repository for reusable CI compilation testing workflows and containers used across ACCESS-NRI supported projects.
 
 This repository is also responsible for building Docker images used for CI compilation testing.
 
-# Building CI Docker images
+## Overview
+
+This repository contains three overarching CI pipelines:
+
+### Dependency Image Pipeline
+
+This pipeline creates Docker images that contain an install of `spack`, a version of the `access-nri/spack_packages` [repository](https://github.com/ACCESS-NRI/spack_packages), and a set of independent `spack env`ironments that contain all the dependencies for all the model components of a coupled model.
```

Comment:
Choose between "`spack env`" and "spack environments"

Reply:
Resolved as part of https://github.com/ACCESS-NRI/build-ci/pull/100#discussion_r1332523976

Reply:
Good idea to contain a link to the docs, will incorporate that